Parsers/ASimAuthentication/ProductParsers/AuthenticationM365D.yaml

```diff
@@ -0,0 +1,131 @@
+       EventOriginalResultDetails=FailureReason 
+       , EventSubType=LogonType
+    | extend 
+    //
+       // ----  Untouched
+       TimeGenerated
+    // 
+      // ----  Event
+       , EventCount=int(1)
+       , EventStartTime=TimeGenerated
+       , EventEndTime=TimeGenerated
+       , EventOriginalType = EventSubType // Previously:LogonType
+       , EventProduct='M365 Defender for EndPoint'
+      ,  EventResult = case(ActionType =='LogonSuccess', 'Success'
+                            , ActionType=='LogonFailed', 'Failure'
+                            , ActionType=='LogonAttempted', 'NA' //// Not sure what this is
+                            , 'NA')
+      , EventSchemaVersion='0.1.0'
+      , EventType='Logon'
+      , EventVendor ='Microsoft'
+    //    -   No Data
+    //  EventOriginalUid 
+    //  EventProductVersion 
+    //  EventReportUrl
+    //  EventMessage 
+    //  
+    //  
+    //  ----   Target and Actor Users
+     | project-rename 
+         TargetUserId=AccountSid
+       , ActorUserId =InitiatingProcessAccountSid
+       , ActorUserUpn=InitiatingProcessAccountUpn
+       , ActorUserObjectId=InitiatingProcessAccountObjectId
+      | extend 
+         TargetUserIdType ='UserSid'
+       , TargetUsername = strcat(AccountDomain,'\\',AccountName)
+       , TargetUsernameType='Windows'
+    //    -  No Data
+      // TargetUserType 
+    // 
+    // ActingAppName 
+    // ActingAppType 
+    // ActorSessionId 
+      , ActorUserIdType='SID'
+    // ActorUserType 
+      , ActorUsername=coalesce(ActorUserUpn, strcat(InitiatingProcessAccountDomain,'\\',InitiatingProcessAccountName)) // InitiatingProcessAccountName
+      , ActorUsernameType=case(isnotempty( ActorUserUpn), 'Upn/Email'
+                                , isnotempty(InitiatingProcessAccountDomain), 'Windows'
+                                , 'Simple')
+     //, AdditionalFields:already exists
+      | project-rename 
+        DvcHostname=DeviceName
+      , LogonProtocol=Protocol
+      , TargetDvcId=DeviceId
+      , TargetDvcIpAddr=RemoteIP
+      , OriginalEventUid=ReportId
+    // No Data
+    // DvcIpAddr 
+    // HttpUserAgent 
+    // LogonMethod 
+    //
+      , SrcDvcHostname=RemoteDeviceName 
+      // SrcDvcHostnameType == Requires calculation. not sure it is worth it
+    // SrcDvcId 
+    // SrcDvcIpAddr 
+    // SrcDvcOs 
+    // SrcDvcType 
+    // SrcGeo fields 
+    //
+    // TargetAppId 
+    // TargetAppName 
+    // TargetAppType 
+    // TargetDvc
+    //
+    //, ActingProcessCommandLine = InitiatingProcessCommandLine
+    //, ActingProcessCreationTime=InitiatingProcessCreationTime
+    , ActingProcessPath=InitiatingProcessFolderPath
+    , ActingProcessId=InitiatingProcessId
+    //, ActingProcessMD5=InitiatingProcessMD5
+    //, ActingProcessSHA1=InitiatingProcessSHA1
+    //, ActingProcessSHA256= InitiatingProcessSHA256
+      | extend 
+          ActingProcessFileName=iff(ActingProcessPath hassuffix InitiatingProcessFileName
```

Comment:
We don't have such a field, we have ActingProcessName

Reply:
yes, looked at ProcessEvent for reference.